File: Ejercicios Propuestos 1/Ejercicio 13/Ejercicio13.java
import java.math.BigInteger;
public class Ejercicio13 {
    public static void main(String[] args) {
        long suma = 0;
        BigInteger producto = BigInteger.ONE;
        for (int i = 20; i <= 400; i += 2) {
            suma += i;
            producto = producto.multiply(BigInteger.valueOf(i));
        }
        System.out.println("Resultados para números pares entre 20 y 400:");
        System.out.println("Suma: " + suma);
        System.out.println("Producto: " + producto);
    }
}

File: Ejercicios Propuestos 1/Ejercicio 13/Ejercicio 13.jl
suma = 0
producto = BigFloat(1.0)

for i in 20:2:400
    global suma += i
    global producto *= i
end

println("Resultados:")
println("La suma de números pares entre 20 y 400 es: ", suma)
println("El producto de números pares entre 20 y 400 es: ", producto)
